webapp/src/components/App/Singlestudy/explore/common/OutputFilters.tsx

```diff
@@ -0,0 +1,64 @@
+import { FieldPath } from "react-hook-form";
+import { useTranslation } from "react-i18next";
+import SelectFE from "../../../../common/fieldEditors/SelectFE";
+import Fieldset from "../../../../common/Fieldset";
+import { ControlPlus } from "../../../../common/Form";
+
+type FilteringType = "hourly" | "daily" | "weekly" | "monthly" | "annual";
```

Comment:
Add an utils.ts file in Modelization and move the type in it, because is already duplicate 3 times.

Use `time_step: Exclude<FilteringType, "monthly" | "annual">;` in "Modelization/BindingConstraints/BindingConstView/utils.ts"